starter-code/css/main.css

```diff
@@ -5,4 +5,86 @@ Text: 1A1A1A
 Green: #00B172
 White: #FFF
 
+
+The page is split into 4 sections.
+
+Navbar
+The navbar should be position: fixed.
+Float the image left, and float a ul
+with the links right.
+Large image background with text
+Check out this guide on centering things.
+What's on Spotify section
+It looks like the divs take up about a third
+of the container each. How can you represent this in code?
+Green Section
+It looks like we have 2 main pieces,
+a div with the text content,
+and the image of the Spotify player.
+Position these according to the surrounding green div.
+Position the Spotify logo absolutely
+according to the Green div.
+Starter code
+The starter-code contains all the files,
+images, and text content needed to create the page.
+The text is in the index.html, and a css file is
+created in main.css with colors.
 */
+
```

Comment:
As you may have notice there is default styling that is apply by your browser, here it's a margin, what you can do to remove it and get rid of the white space at the top is to reset those value, like this : 

```css

* {
  margin: 0 auto;
  padding: 0 auto;
  font-family: MOntserrat;
}

```

You can also add a default font :)